src/utils/pipes/ssi-types-parser.pipe.ts

```diff
@@ -80,12 +69,12 @@ export class SsiTypesParserPipe
   private transformVerifiableCredential(verifiableCredential: VerifiableCredentialDto<any>): TypedVerifiableCredential {
     try {
       const originalVerifiableCredential = { ...verifiableCredential }
-      const type = getTypeFromSelfDescription(verifiableCredential)
-      const { credentialSubject } = verifiableCredential
-      delete verifiableCredential.credentialSubject
+      const type = getTypeFromSelfDescription(originalVerifiableCredential)
+      const { credentialSubject } = originalVerifiableCredential
+      delete originalVerifiableCredential.credentialSubject
```

Comment:
I guess we need to change the name a bit. original suggests it is unaltered, whilst we are manipulating it here

Reply:
yeah, I agree. Made it other way around